src/import/sst_service.rs

```diff
@@ -25,7 +25,7 @@ use tikv_util::time::Instant;
 use sst_importer::import_mode::*;
 use sst_importer::metrics::*;
 use sst_importer::service::*;
-use sst_importer::{Config, Error, SSTImporter};
+use sst_importer::{Config, Error, SSTImporter, error_inc};
```

Comment:
`error_inc` seems unused in this file.

Reply:
It been used in macro `send_rpc_response!`